[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-1 (#114270)
openshift/openshift-docs: modules/compliance-priorityclass.adoc

[id="compliance-priorityclass_{context}"]
= Setting `PriorityClass` for `ScanSetting` scans

[role="_abstract"]

> In large scale environments, the default `PriorityClass` object can be too low to guarantee Pods execute scans on time. For clusters that must maintain compliance or guarantee automated scanning, it is recommended to set the `PriorityClass` variable to ensure the Compliance Operator is always given priority in resource constrained situations.

Reviewer: Length
```suggestion
In some clusters, the default `PriorityClass` object can be too low to guarantee pods execute scans on time. To maintain compliance or guarantee automated scanning, you can set the `PriorityClass` variable to ensure the Compliance Operator is always given priority in resource constrained situations.
```
